.github/workflows/check.yml

```diff
@@ -74,6 +74,62 @@ jobs:
           flags: tests
           name: ${{ matrix.py }} - ${{ matrix.os }}
 
+  test-android:
```

Comment:
I don't think these should be new test targets in the CI. IMHO this should be just a test suite.

Reply:
If i understood correctly, it should test on ubuntu-latest both linux and android behaviors?

Reply:
I'd say UNIX in general, either macos or ubunut will work, not?